src/frontend/src/pages/PIAForm/BannerStatus/index.tsx

```diff
@@ -2,21 +2,28 @@ import { useState, useEffect } from 'react';
 import { IBannerStatusProps } from './interfaces';
 import Callout from '../../../components/common/Callout';
 import { PiaStatuses } from '../../../constant/constant';
-import { isMPORole } from '../../../utils/helper.util';
+import { isMPORole, getGUID } from '../../../utils/helper.util';
```

Comment:
Could we switch to the new role check pattern? Ideally, I think we want to transition to this way of checking the user role because it is extendable if there are added roles in the future.

Reply:
i think I switch to the new pattern